file-formats/ront/type/zif_aff_ront_v1.intf.abap

```diff
@@ -17,39 +17,47 @@ INTERFACE zif_aff_ront_v1
       configuration_object TYPE c LENGTH 2 VALUE `CO`,
       "! <p class="shorttext">Dependent Object</p>
       "! Dependent object
-      dependent_object TYPE c LENGTH 2 VALUE `DO`,
+      dependent_object     TYPE c LENGTH 2 VALUE `DO`,
       "! <p class="shorttext">Hierarchy Object</p>
       "! Hierarchy object
-      hierarchy_object TYPE c LENGTH 2 VALUE `HO`,
+      hierarchy_object     TYPE c LENGTH 2 VALUE `HO`,
     END OF co_category.
 
   TYPES:
     "! <p class="shorttext">SAP Object Type</p>
     "! SAP object type
     BEGIN OF ty_main,
       "! $required
-      format_version   TYPE zif_aff_types_v1=>ty_format_version,
+      format_version                TYPE zif_aff_types_v1=>ty_format_version,
 
       "! <p class="shorttext">Header</p>
       "! Header
       "! $required
-      header           TYPE zif_aff_types_v1=>ty_header_60_cloud,
+      header                        TYPE zif_aff_types_v1=>ty_header_60_cloud,
 
       "! <p class="shorttext">Type Category</p>
       "! The category of the SAP Object Type
       "! $required
       "! $values {@link zif_aff_ront_v1.data:co_category}
       "! $default {@link zif_aff_ront_v1.data:co_category.business_object}
-      type_category    TYPE c LENGTH 2,
+      type_category                 TYPE c LENGTH 2,
 
       "! <p class="shorttext">Name</p>
       "! The name of the SAP Object Type.
       "! $required
-      name             TYPE c LENGTH 30,
+      name                          TYPE c LENGTH 30,
 
       "! <p class="shorttext">Object Type Code</p>
       "! The object type code uniquely identifies the SAP Object Type.
-      object_type_code TYPE c LENGTH 5,
+      object_type_code              TYPE c LENGTH 5,
+
+      "! <p class="shorttext">Interface Behavior Definition</p>
+      "! The interface behavior definition related to the SAP Object Type.
+      interface_behavior_definition TYPE c LENGTH 30,
```

Comment:
We struggled with the term Interface Behavior Definition. Is it a special interface or a special behavior definition? For us, it sounds very technical. But we may need more input on this field.